src/mtcp/mtcp_restart.c

```diff
@@ -302,7 +303,8 @@ MTCP_PRINTF("Attach for debugging.");
 }
 
 NO_OPTIMIZE
-static void restore_brk(VA saved_brk, VA restore_begin, VA restore_end)
+static void restore_brk(VA saved_brk, VA restore_begin, VA restore_end,
+                        RestoreInfo * rinfo_ptr)
```

Comment:
It seems like we never use this new extra argument -- `rinfo_ptr` -- in the `restore_brk()` function. It's aliased to the global `rinfo` object, anyway. I think we should remove the extra argument.